README.md

```
@@ -77,6 +77,45 @@ yarn lint
 yarn lint:style
 ```
 
+## Worker
+
+Collection and the HTTP API run as one Cloudflare Worker. Its code lives under `worker/`, separate from the frontend in `src/`, and `wrangler.toml` at the root configures it.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)